src/index.ts

```diff
@@ -1,9 +1,21 @@
 declare module '@webcomponents/custom-elements';
 
-(async function () {
-  if (!('customElements' in self)) {
-    await import('@webcomponents/custom-elements');
-  }
-
+if (typeof PRERENDER === 'boolean') {
+  // @ts-ignore
+  window.customElements = {
+    define() {},
+  };
```

Comment:
Is this because it's running in jsdom?

Reply:
yup.

Reply:
I'll add a comment.

Reply:
I moved all this shimming into prerender-loader where it belonged. Now our index is super clean!